addons/payment_authorize/models/authorize_request.py

```diff
@@ -226,11 +226,15 @@ def auth_and_capture(self, token, amount, reference):
                 'x_response_reason_text': response.get('err_msg')
             }
 
-        return {
+        result = {
             'x_response_code': response.get('transactionResponse', {}).get('responseCode'),
             'x_trans_id': response.get('transactionResponse', {}).get('transId'),
             'x_type': 'auth_capture'
         }
+        errors = response.get('transactionResponse', {}).get('errors')
+        if errors:
+            result['x_response_reason_text'] = '\n'.join([e.get('errorText') for e in errors])
```

Comment:
Nitpicking
```suggestion
            result['x_response_reason_text'] = '\n'.join(e.get('errorText') for e in errors)
```
My comment was scrapped somehow so here it is.